conda/recipe/meta.yaml

```diff
@@ -35,6 +35,7 @@ build:
    - BUILD_LIBTIFF
    - BUILD_NVOF
    - BUILD_NVDEC
+   - BUILD_LIBSND
```

Comment:
You need to add https://anaconda.org/conda-forge/libsndfile as a build and run dependency as well here.

Reply:
Will be addressed here https://github.com/NVIDIA/DALI/pull/1483